website/blog/2021-02-22-7.13.0.md

```diff
@@ -0,0 +1,255 @@
+
+Babel now supports transforming the [Records and Tuples](https://github.com/tc39/proposal-record-tuple) ECMAScript proposal, which brings immutable data structures and structural equality to JavaScript, and parsing the [Module Blocks](https://github.com/tc39/proposal-js-module-blocks) proposal.
+
+Additionally, we added support for some new Flow and TypeScript features.
+
+You can read the whole changelog [on GitHub](https://github.com/babel/babel/releases/tag/v7.12.0).
+
+<!-- truncate -->
+
+### Funding Update
+
+We have joined the "GitHub Sponsors for organizations" program, so you can now sponsor us directly through GitHub 😊.
+
+<iframe src="https://github.com/sponsors/babel/button" title="Sponsor Babel on GitHub" height="35" width="116" style="border: 0; margin: auto; display: block"></iframe>
+
+These funds are used to support our team's (currently one full-time and three part-time maintainers) continued efforts in improving stability and developing new features.
+
+Both our donations and expenses are openly tracked through our [Open Collective](https://opencollective.com/babel) page, where we will also track donations coming from GitHub Sponsors.
+
+We welcome donations from both individual and companies: if your company is interested in becoming a Gold tier sponsor ($1k/month) and wants to discuss more, please reach out to [team@babeljs.io](mailto:team@babeljs.io)!
+
+## Highlights
+
+### Top-level `targets` option ([#12189](https://github.com/babel/babel/pull/12189), [RFC](https://github.com/babel/rfcs/pull/2))
+
+The `targets` option is the true power behind `@babel/preset-env`: it allows users the ability to specify their target environments which allows the preset to automatically choose which syntax to transform and what polyfills to inject.
+
+Since releasing `@babel/preset-env`, we've learned that plugins _themselves_ can also benefit from knowing your targets. Currently, this can a bit cumbersome, since you'd have to specify your targets twice (for example, if you are using our [new polyfill plugins](https://github.com/babel/babel-polyfills). By introducing `targets` as a top level option, you now only have to specify your targets once:
+
+<table>
+<tr>
+<th>Old configuration</th>
+<th>New configuration</th>
+</tr>
+<tr>
+<td>
+
+```json
+{
+  "presets": [
+    ["@babel/preset-env", {
+      "targets": ">1%, not ie 11"
+    }]
+  ],
+  "plugins": [
+    ["polyfill-es-shims", {
+      "targets": ">1%, not ie 11"
+    }]
+  ]
+}
+```
+
+</td>
+<td>
+
+```json
+{
+  "targets": ">1%, not ie 11",
+  "presets": ["@babel/preset-env"],
+  "plugins": ["polyfill-es-shims"]
+}
+
+
+
+
+
+
+
+```
+
+</td>
+</tr>
+</table>
+
+We recommended converting your Babel config to use the new top-level `targets` option since:
+
+1. It has better handling of the [`esmodules: true`](TODO: Link) target. (it's intersected with the other targets, rather than replacing them)
```

Comment:
```suggestion
1. It has better handling of the [`esmodules: true`](https://babel.dev/docs/en/options#targetsesmodules) target. (it's intersected with the other targets, rather than replacing them)
```

This will link to https://deploy-preview-2416--babel.netlify.app/docs/en/options#targetsesmodules